### ISSUE TITLE
Mac Firefox 120.0.1 issue

### ISSUE THREAD

**Issue by polypx** [putdotio/putio-webextension#4]
This has been going on for the last couple of weeks.
1. The put.io item in the right click drop down doesn't work.  ie. it's there, but does nothing to a magnetic link
2. I turn the extension off, and back on again (in Firefox extensions settings)
3. The put.io item in right click works once, on a magnetic link
4. The put.io item no longer works, until I once again go to 2. and turn it off and on again

In other words, I only get one use out of every activation.   

This is recent, it used to work correctly.

**Comment by cenkalti** [comment]
Yes, I can confirm this is happening for me as well. It seems that Firefox addons on manifest v3 is unloaded on idle. Unfortunately, Firefox doesn't support service workers yes. I'll revert back to manifest v2. Just need to find a way to maintain Chrome and Firefox in a single implementation.

**Comment by cenkalti** [comment]
Hi @polypx . We've reverted back the manifest v3 version. You should see latest version as 2.3.3 in addon store. Please let us know if this does not solve the issue for you. Thanks for reporting this.

**Comment by polypx** [comment]
Brilliant, seems to be working fine now, thanks!

**Comment by KAMiKAZOW** [comment]
> You should see latest version as 2.3.3 in addon store.

"Last updated 4 years ago (Jun 12, 2020)" ???

![image](https://github.com/putdotio/putio-webextension/assets/3259076/0b680dbf-c33c-44a9-876d-d1546e824090)
